docs/news/index.md

```diff
@@ -1,69 +1,51 @@
+[**Voting and Governance in Security Tokens**](https://medium.com/coinmonks/voting-and-governance-in-security-tokens-1e3d041dabb8) | [**Crypto Commons — from tragedy to wealth of the commons**](https://blog.usejournal.com/crypto-commons-da602fb98138) | |
 :-----------:|:-----------:|:-----------:|
-| [Aragon](aragon.md) |[Governance](governance.md)|[Aragon](aragon.md)|
-| [<img src=https://cdn-images-1.medium.com/max/2000/1*yNk4bvKU5Wmfrtx5t22R-Q.png>](https://blog.aragon.one/aragon-network-on-a-path-towards-a-digital-jurisdiction-752ccf4b9d5c) |[<img src=https://4.bp.blogspot.com/-QYTO1m6eCWY/UKv0dYaWYFI/AAAAAAAAAC8/3ryN1b5-OFE/s1600/school-of-athens2.jpg>](https://medium.com/betoken/we-need-to-formalize-governance-research-using-daos-e9393da427f3)| [<img src=https://cdn-images-1.medium.com/max/2000/1*o7yRBPfaXV7lGQRbQ5eXmQ.jpeg>](https://blog.aragon.one/aragon-launches-survey-app-on-mainnet-ed5eefeb66f5) |
-| Author [Luke Duncan](https://blog.aragon.one/@lkngtn) on May 25 |Author [Zefram Lou](https://medium.com/@zeframlou) on May 28| Author [Tatu Kärki](https://blog.aragon.one/@Smokyish) on May 31 |
-| **Launch of a new edition of the Aragon Network whitepaper and our research forum**. The initial version of the Aragon Network whitepaper was published over a year ago. Since then we have made tremendous progress in making Decentralized Autonomous Organizations a reality. |As a human race we started taking governance seriously during the Renaissance period. We started to use our rationality to analyze our political systems, and developed a theoretical framework to think about governance systems, namely modern political science. Turns out it was pretty useful.| **First Aragon app launched on Ethereum Mainnet for community sentiment signaling**. Today we are releasing an entirely new step for community members to signal their sentiment on the project and igniting the next level of participation in the project for all ANT holders. Survey app is the fourth app developed by our team in addition to the Token Manager, Voting & Finance apps seen previously in Aragon Core v0.5. And the first one to launch on the Ethereum Mainnet.|
-| [Read More](https://blog.aragon.one/aragon-network-on-a-path-towards-a-digital-jurisdiction-752ccf4b9d5c) |[Read More](https://medium.com/betoken/we-need-to-formalize-governance-research-using-daos-e9393da427f3)| [Read More](https://blog.aragon.one/aragon-launches-survey-app-on-mainnet-ed5eefeb66f5) |
+[_Governance_](governance.md) | [_Governance_](governance.md) | |
+[<img src="https://cdn-images-1.medium.com/max/800/1*F7oHt_loAWz4EACe_SqRgQ.jpeg">](https://medium.com/coinmonks/voting-and-governance-in-security-tokens-1e3d041dabb8) | [<img src="https://cdn.britannica.com/668x448/09/134309-004-ABF9FE28.jpg">](https://blog.usejournal.com/crypto-commons-da602fb98138) | |
```

Comment:
Last, empty, column could use the `../../images/monthly_no_image.png` image and the `Submit News!` link.